### ISSUE TITLE
Work out how we can place the css files in child folders of /imports and still have them be recognised

### ISSUE THREAD

**Issue by tomRedox** [tomRedox/simpleCRM#10]

**Comment by StanLindsey** [comment]
Although not specifically following mantra, css for each component is suggested to go alongside the .jsx. It would follow that primary CSS, for say the layout, would go with the Layout.jsx as well? 

https://kadirahq.github.io/mantra/#sec-components

Thoughts?

**Comment by tomRedox** [comment]
I totally agree.  What I should have made clear is that doing that seems to stop Meteor seeing them in 1.3; they are then just ignored.  I guess that's because everything in the imports folder is lazy loaded now and css needs to be loaded at the outset?

I've tried adding them as imports to the main.less, and also creating a main.css that includes:

```
@import "./imports/ui/components/npm-css-copies/flipCOPY.css";
@import "./imports/ui/components/npm-css-copies/react-selectCOPY.min.css";
@import "./imports/ui/components/npm-css-copies/s-alert-defaultCOPY.css";
```

but neither of those approaches works.
